File: 1859-sorting-the-sentence/1859-sorting-the-sentence.cs
public class Solution {
    public string SortSentence(string s) {
        List<string> list = new List<string>();
        
        list = s.Split(" ").ToList();
        string[] ar = new string[list.Count];
        
        foreach(var word in list){
            int i = int.Parse(word[word.Length-1].ToString());
            
            ar[i-1] = word.Substring(0, word.Length-1);
        }
        
        return String.Join(" ", ar);
    }
}

File: 1859-sorting-the-sentence/1859-sorting-the-sentence.py
class Solution:
    def sortSentence(self, s: str) -> str:
        list = s.split(" ")
        
        res = [None for i in range(len(list))]
        
        for st in list:
            res[int(st[-1])-1]= st[:-1:]
            
        
        
        return " ".join(res)
        